### PR TITLE
Update var to const/let

### PR DESCRIPTION
**Pull request by victoria-kaihe**
Updated instances of var to const in script.js to open PR

### CODE REVIEW

**Comment by victoria-kaihe** [comment]
I've been through all the docs in this repo and hopefully I've got all the vars. Have updated the label to "awaiting-review".

File: README.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ the notes and coins can be represented as:
 this can be represented as an Array:
 
 ```javascript
-var coins = [5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 5, 2, 1]
+let coins = [5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 5, 2, 1]
```
Comment (nelsonic) [inline README.md]: the coins won't change so you can have them as a `const` too. 😉

File: lib/change-calculator.js [hunk 2]
```diff
@@ -8,18 +8,18 @@
  */
  module.exports = function calculateChange(totalPayable, cashPaid) {
 
-   var coins = [5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 5, 2, 1];
-   var change = [];
-   var length = coins.length;
-   var remaining = cashPaid - totalPayable;          // we reduce this below
+   let coins = [5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 5, 2, 1];
+   let change = [];
+   let length = coins.length;
```
Comment (nelsonic) [inline lib/change-calculator.js]: `length` will also never change as it's the length of a _fixed_ Array.

**Review by nelsonic — COMMENTED** [review]
@victoria-kaihe looks good. (_thanks for opening this PR!_)
A couple of instances of `let` could be swapped for `const` (_whenever something isn't going to change, you might as well signal this intent to your fellow programmer..._)

**Comment by victoria-kaihe** [comment]
@nelsonic thanks for reviewing my changes! I've updated those lets to consts and pushed the changes for review

**Review by nelsonic — APPROVED** [review]
@victoria-kaihe looks great. thanks for taking the time to create this PR! 🎉

**Pull request merged**